### PR TITLE
updated publish_pypi job in config.yml

### PR DESCRIPTION
**Pull request by yodragon-github**
merging

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ workflows:
                 - develop
       - pypi_publish:
           requires:
-            - build_test
+            - test_pypi_publish
```
Comment (ravindra-rao) [inline .circleci/config.yml:64]: why did you change this?

Comment (yodragon-github) [reply .circleci/config.yml:64]: updating the requirement for publish_pypi job

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 setuptools.setup(
     include_package_data=True, 
     name='yopycalc', 
-    version='0.0.2.1', 
+    version='0.0.2.2',
```
Comment (yodragon-github) [inline setup.py:7]: changing version number?

Comment (ravindra-rao) [inline setup.py:7]: changed version number?

Comment (yodragon-github) [reply setup.py:7]: yes

**Review by ravindra-rao — CHANGES_REQUESTED** [review]
change it

File: setup.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 setuptools.setup(
```
Comment (ravindra-rao) [inline setup.py:4]: change it

**Review by ravindra-rao — CHANGES_REQUESTED** [review]
hi

File: setup.py (hunk 3 above)
Comment (yodragon-github) [reply setup.py:4]: okay

**Review by ravindra-rao — APPROVED** [review]
